src/View/View.php

```diff
@@ -685,6 +685,36 @@ public function element(string $name, array $data = [], array $options = []): st
         throw new MissingElementException([$name . $this->_ext, $elementName . $this->_ext], $paths);
     }
 
+    /**
+     * Includes a template fragment, specific to the controller's folder.
+     *
+     * This method allows you to include a fragment of the template, which is placed in the folder
+     * of the current controller instead of the `element` folder. If the fragment is used across
+     * multiple controllers, you can specify an absolute path starting with `/`.
+     *
+     * @param string $name The name of the fragment template to include.
+     * @param array $data The data to pass into the fragment template.
+     * @param array $options Additional options for rendering the fragment.
+     * @return string Rendered HTML content of the fragment.
+     */
+    public function fragment(string $name, array $data = [], array $options = []): string
+    {
+        // Ensure data is always an array
+        if (!is_array($data)) {
+            $data = ['data' => $data]; // Wrap in array if needed
+        }
+
+        // Construct the path for the fragment
+        if ($name[0] == '/') {
+            $path = "..{$name}";
+        } else {
+            $path = "..{$this->name}/{$name}";
```

Comment:
Using the `$name` is not correct IMO. For e.g. if the current controller name is `Admins` but the template path is set to `Users` then I would expect it to look for the fragment under `Users` folder.

The View class is also used for rendering emails and cell where the `$name` is irrelevant/empty.